### PR TITLE
le-auto shell style improvements

### PR DESCRIPTION
**Pull request by erikrose**
Ported from #1751 to the new le-auto

### CODE REVIEW

**Comment by Arthur2e5** [comment]
Looks good to me. Just one more thing…

File: letsencrypt-auto-source/letsencrypt-auto.template [hunk 1]
```diff
@@ -91,18 +97,15 @@ ExperimentalBootstrap() {
...
+    command -v "$LE_PYTHON" > /dev/null && break
+  done
+  if [ "$?" != "0" ]; then
+    echo "Cannot find any Pythons; please install one!"
     exit 1
   fi
+  export LE_PYTHON
 
   PYVER=`"$LE_PYTHON" -V 2>&1 | cut -d" " -f 2 | cut -d. -f1,2 | sed 's/\.//'`
   if [ $PYVER -lt 26 ]; then
```
Comment (Arthur2e5) [inline letsencrypt-auto-source/letsencrypt-auto.template]: Should this be `-le` (<=) instead?

Comment (Arthur2e5) [inline letsencrypt-auto-source/letsencrypt-auto.template]: Oh fa49976…… I see.

**Comment by pde** [comment]
A slightly patched `test_letsencrypt_auto_certonly_standalong` (I added --debug --version to the --os-packages-only call):
```
0 ubuntu15.10 pass
1 ubuntu15.04LTS pass
2 ubuntu14.04LTS pass
3 ubuntu14.04LTS_32bit pass
5 debian8.1 pass
6 debian7.8.aws.1 pass
7 debian7.8.aws.1_32bit pass
8 amazonlinux-2015.09.1 pass
9 amazonlinux-2015.03.1 pass
10 RHEL7 pass
11 fedora23 pass
12 centos7 pass
13 centos6 pass
```

**Pull request merged**